src/utils/hooks/api/use-get-available-vaults.tsx

```diff
@@ -0,0 +1,77 @@
+import { CollateralCurrencyExt, CurrencyIdLiteral } from '@interlay/interbtc-api';
```

Comment:
Bit confused why this is here - is it a merging issue? There is now a `vaults` subdirectory in utils/hooks/api which is where this hook should be.